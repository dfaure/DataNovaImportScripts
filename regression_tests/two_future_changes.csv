14993A;NEZEL AP;2020-11-27;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-11;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-26;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-13;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-06;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-25;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-23;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-02;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-24;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-04;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-05;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-13;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-23;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-14;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-03;16:00-19:00;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-17;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-16;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-08;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-09;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-15;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-30;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-12;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-22;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-28;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-10;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-10;16:00-19:00;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-19;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-07;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-01;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-21;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-12;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-09;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-18;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-16;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-29;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-07;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-19;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-20;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-18;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-28;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-17;16:00-19:00;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-11-11;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-21;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-12;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-20;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-18;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-01;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-16;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-08;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-15;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-05;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-31;16:00-19:00;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-24;16:00-19:00;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-30;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-22;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-03;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-29;15:00-17:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-01;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-09;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-26;09:00-12:30;;;;;;;11:30;11:30;11:30;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-06;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-13;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-11;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2020-12-14;09:00-12:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-04;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-02;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-15;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-25;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-04;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-22;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-10;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-14;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-30;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-23;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-08;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-05;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-19;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-21;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-26;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-27;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-09;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-29;FERME;;;;;;;;;;;FERME;;;;;;;
14993A;NEZEL AP;2021-01-28;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-06;09:00-12:00;;;;;;;11:30;11:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-02;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
14993A;NEZEL AP;2021-02-11;15:00-18:30;;;;;;;16:30;16:30;;;FERME;;;;;;;
