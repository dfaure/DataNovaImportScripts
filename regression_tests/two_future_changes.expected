14993A|NEZEL AP|Mo,Sa 09:00-12:00; 2020 Mo,Sa 09:00-12:30; Tu 15:00-18:30; 2020 Tu 15:00-17:30; Th 15:00-17:30; PH off
